### DISCUSSION TITLE
Update the rustdesk docker image

### DISCUSSION THREAD

**Discussion by xiaobaixuejava** [rustdesk/rustdesk#7905] [General]
When do you plan to update the docker image? The last docker image was three months ago

**Comment by dimejo** [comment]
@xiaobaixuejava:
Are you referring to the docker images of RustDesk server? I don't see any official images of the client.

**Reply by xiaobaixuejava** [reply]
https://hub.docker.com/r/rustdesk/rustdesk-server  
This is an image. I haven't updated it for a long time. I don't know if it's an official image. If so, can I update it to support the latest features?

**Comment by xlionjuan** [comment]
Because there has no update 

https://github.com/rustdesk/rustdesk-server
